zqd/space/archivesubspace.go

```diff
@@ -6,51 +6,44 @@ import (
 
 	"github.com/brimsec/zq/pkg/nano"
 	"github.com/brimsec/zq/zqd/api"
-	"github.com/brimsec/zq/zqe"
 )
 
 type archiveSubspace struct {
 	spaceBase
 	parent *archiveSpace
 }
 
-func (s *archiveSubspace) Info(ctx context.Context) (api.SpaceInfo, error) {
+func (s *archiveSubspace) Info(ctx context.Context) (info api.SpaceInfo, err error) {
 	sum, err := s.store.Summary(ctx)
 	if err != nil {
-		return api.SpaceInfo{}, err
+		return info, err
```

Comment:
Nit: I think this switch to named result parameters makes the behavior here less clear.

Reply:
fair enough